vecgeom.spec

```diff
@@ -1,16 +1,16 @@
-### RPM external vecgeom v1.2.11
+### RPM external vecgeom v2.0.0-rc.5
 ## INCLUDE compilation_flags
 ## INCLUDE compilation_flags_lto
 ## INCLUDE cpp-standard
 ## INCLUDE microarch_flags
 
-%define tag 47dd602df7074fcc78036e93cd639ae6270207fd
-Source: git+https://gitlab.cern.ch/VecGeom/VecGeom.git?obj=master/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
+%define vecgeom_version %(echo %{realversion} | sed 's|^v||;s|-.*||')
```

Comment:
@akritkbehera , this line is not needed, looks like we already have this at https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_15_1_X/g4/vecgeom.spec#L14